hardware/spectrometer/oceanoptics_spectrometer.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+This module controls spectrometers from Ocean Optics Inc.
+All spectrometers supported by python-seabreeze should work.
+
+Do "conda install -c poehlmann python-seabreeze"
+before using this module.
+
+Qudi is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Qudi. If not, see <http://www.gnu.org/licenses/>.
+
+Derived from the pyPL project (https://github.com/kaseyrussell/pyPL)
+Copyright 2010 Kasey Russell ( email: krussell _at_ post.harvard.edu )
+
+Copyright (c) the Qudi Developers. See the COPYRIGHT.txt file at the
+top-level directory of this distribution and at <https://github.com/Ulm-IQO/qudi/>
+
+"""
+
+from core.module import Base, ConfigOption
+from interface.spectrometer_interface import SpectrometerInterface
+import numpy as np
+import seabreeze.spectrometers as sb
+
+
+
+class OceanOptics(Base, SpectrometerInterface):
+    """ Hardware module for reading spectra from the WinSpec32 spectrometer software.
```

Comment:
I'm not sure but does the _Ocean Optics_ spectrometer use _WinSpec32_ or is this just a copy-paste leftover from another module?

Reply:
Was an error and is fixed, Thanks.